src/components/editor-content/editor-content.tsx

```diff
@@ -745,26 +744,54 @@ const EditorContent = () => {
   * Renders vacation comment box
   */
   const renderVacationCommentBox = () => (
-    <TextField
-      id="outlined-multiline-flexible"
-      multiline
-      maxRows={5}
-      label={ strings.editorContent.leaveAComment }
-      variant="outlined"
-      value={textContent}
-      onChange={handleVacationCommentContent}
-    />
+    <Box>
+      <Typography variant="h4">
+        { strings.editorContent.amountOfChosenVacationDays }
+        { renderVacationDaysSpent() }
+      </Typography>
+      <TextField
+        id="outlined-multiline-flexible"
+        multiline
+        maxRows={ 5 }
+        label={ strings.editorContent.leaveAComment }
+        variant="outlined"
+        value={textContent}
+        onChange={ handleVacationCommentContent }
+      />
+    </Box>
+    
   );
 
   /**
   * Handle vacation apply button
+  * Sends vacation request to database
   */
-  const handleVacationApplyButton = () => {
-    // TODO: send vacation request to database
-    return (
-      // eslint-disable-next-line no-console
-      console.log(`this is START DATE ${selectedVacationStartDate} and this is END DATE${selectedVacationEndDate} and this is TEXT CONTENT ${textContent}. Vacation type ${vacationType}`)
-    );
+  const applyForVacation = async () => {
+    const newRequest: VacationRequest = {
+      person: 123456,
+      startDate: selectedVacationStartDate,
+      endDate: selectedVacationEndDate,
+      type: vacationType as VacationType,
```

Comment:
This won't need to be cast if the state is typed only as VacationType as suggested in previous comment.